db/migrate/20210201170927_add_alt_to_content_assets.rb

```diff
@@ -0,0 +1,5 @@
+class AddAltToContentAssets < ActiveRecord::Migration[6.1]
+  def change
+    add_column :content_assets, :alt, :string
```

Comment:
Should a `null: false` be added to this column?

Reply:
check, had to add default though as well for rubocop